install/docker/judge/Dockerfile

```
@@ -5,24 +5,14 @@ RUN apt-get -qq update && \
-    rustc \
-    haskell-platform \
-    libjavascriptcoregtk-4.0-bin \
-    golang \
-    ruby \
-    mono-runtime \
-    mono-mcs
+    openjdk-17-jdk-headless
 
 ADD ./entrypoint.sh /root/entrypoint.sh
-ADD ./judge.yaml /root/judge.yaml
+ADD ./judge.yaml /root/.hydro/judge.yaml
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)